### DISCUSSION TITLE
extension type inference warning

### DISCUSSION THREAD

**Discussion by greatbooms** [prisma/prisma#21791] [Q&A]
### Question

I am using the extends function for the read replica function in nestjs. So I found this article https://www.prisma.io/blog/read-replicas-prisma-client-extension-f66prwk56wow and created a usable service. The service doesn't seem to be doing the type inference correctly, unlike the default connection. I would like to resolve this warning, can you tell me how?

### Information about Prisma Schema, Client Queries and Environment (optional)

![스크린샷 2023-11-06 오후 7 58 25](https://github.com/prisma/prisma/assets/5744408/f8480b03-4d9f-4b37-935d-ad9bfeeef58b)
![스크린샷 2023-11-06 오후 7 58 17](https://github.com/prisma/prisma/assets/5744408/80d36862-5510-43ed-8145-5e9699ec9468)

**Comment by ludralph** [comment]
Hi @greatbooms 👋 

Can you please share the warnings you were getting to help us understand it better?  Could you also provide a reproducible example we can run and see the warning?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
